packages/server/computation_container/share/BUILD

```diff
@@ -38,19 +38,3 @@ cc_library(
     ],
     visibility = ["//visibility:public"],
 )
-
-cc_library(
-    name = "matrix",
-    hdrs = [
-        "matrix.hpp",
-    ],
-    srcs = [
-        "matrix.cpp"
-    ],
-    deps = [
-        "//share:share",
-        "//server/computation_to_computation_container:server",
-        "@eigen//:eigen",
```

Comment:
Since `Eegen` is only used in `matrix.*`, please delete it together.

https://github.com/acompany-develop/QuickMPC/blob/0c8fb12c77b682e381e7af6fc414990fb6e4b696/packages/server/computation_container/Dockerfile#L22
https://github.com/acompany-develop/QuickMPC/blob/0c8fb12c77b682e381e7af6fc414990fb6e4b696/packages/server/computation_container/WORKSPACE#L44-L49
https://github.com/acompany-develop/QuickMPC/blob/main/packages/server/computation_container/eigen.BUILD

Reply:
[5efd180](https://github.com/acompany-develop/QuickMPC/pull/246/commits/5efd18052df72bf9279f9d378e28c5cfcc2e034e)

Reply:
Thank you! Please delete this too.
https://github.com/acompany-develop/QuickMPC/blob/main/packages/server/computation_container/eigen.BUILD

Reply:
[3eb6fc1](https://github.com/acompany-develop/QuickMPC/pull/246/commits/3eb6fc1837a4f3fde9a0b282964cfff5f5f3653d)